strawberryfields/apps/qchem/dynamics.py

```diff
@@ -250,3 +254,84 @@ def prob(samples: list, excited_state: list) -> float:
         raise ValueError("The excited state must not contain negative values")
 
     return samples.count(excited_state) / len(samples)
+
+
+def sample_tmsv(
+    sq: list, t: float, Ul: np.ndarray, w: np.ndarray, n_samples: int, loss: float = 0.0,
+) -> list:
+    r"""Generate samples for simulating vibrational quantum dynamics with a two-mode squeezed input state.
```

Comment:
```suggestion
    r"""Generate samples for simulating vibrational quantum dynamics with a two-mode squeezed vacuum input state.
```
Note that line width may be too long now